aprotos/01HTML/hm_1.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+    <head>
+   <title>Python 1</title>
+    </head>
+    <body>
+     
+     <h1 align="center">HOW-TO: Программа на Python, Часть 1</h1>
+<ul><em>
+	<li>Программа на Python — часть 1</li>
+<li>Программа на Python — часть 2</li>
+<li>Программа на Python — часть 3 </li></em>
+</ul>
+<h2>Содержание</h2>
+<h3>HOW-TO: Программа на Python, Часть 1</h3>
+
+<ul><ins><li>Наша первая программа</li>
```

Comment:
Each ins tag need to put inside each li tag

Reply:
can u explain why? if we have big (e.g. 100+ list) we must put <ins></ins> in each <li>. It's simplier to put <ins> once all over list, dosn't it?

Reply:
All code will be generated by back-end app, it will be putted by script - related difficulty to markup
And do not break  specification of this construction, using your approach, your colleagues will not understand you